WordPress/src/main/java/org/wordpress/android/localcontentmigration/LocalMigrationContentResolver.kt

```diff
@@ -47,7 +47,11 @@ class LocalMigrationContentResolver @Inject constructor(
         }
     }.let { uriBuilder ->
         with(contextProvider.getContext().contentResolver) {
-            val cursor = query(uriBuilder, entityType, entityId)
+            val cursor = try {
+                query(uriBuilder, entityType, entityId)
+            } catch (e: SecurityException) {
+                return Failure(NullCursor(entityType, e))
+            }
```

Comment:
I wonder if we should use `runCatching` (to be consistent with the pattern in use) 🤔

Reply:
I've added [a branch with this suggestion](https://github.com/wordpress-mobile/WordPress-Android/compare/fix/18627-SecurityException...fix/18627-SecurityException--suggestion-new-error-class).

Reply:
> I wonder if we should use runCatching (to be consistent with the pattern in use) 🤔

Great suggestion 👍 
I've pushed a minor fix https://github.com/wordpress-mobile/WordPress-Android/pull/19468/commits/5554e8485b036f829f496da7945f192522d10192 and merged your branch 🙇 

Ready for another pass :)

Reply:
Oh, right, good catch! 😄